res/xml/preferences_notifications.xml

```diff
@@ -65,4 +51,10 @@
                 android:entries="@array/pref_notification_priority_entries"
                 android:entryValues="@array/pref_notification_priority_values"/>
 
+        <org.thoughtcrime.securesms.components.SwitchPreferenceCompat
```

Comment:
The send 'tok' is present even if this is "off". So while not a "notification" sound we should keep this together.

(Receive "Tok" is working as expected)

Reply:
ahh, i though i've fixed that.

Reply:
i have a fix, one second ...